README.md

```diff
@@ -47,30 +46,28 @@ In this case, the output will be:
 {'00': 513, '11': 511}
 ```
 
-A script is available [here](examples/python/ibmq/hello_quantum.py), where we also show how to
-run the same program on a real quantum computer via IBMQ.
+A script is available [here](examples/python/ibmq/hello_quantum.py), where we also show how to run the same program on a real quantum computer via IBM Quantum.
 
 ### Executing your code on a real quantum chip
 
 You can also use Qiskit to execute your code on a
-**real quantum chip**.
-In order to do so, you need to configure Qiskit for using the credentials in
-your IBM Q account:
+**real quantum chip**. 
+In order to do so, you need to configure Qiskit with credentials for using a quantum computer cloud provider. For example, to connect with IBM Quantum account:
 
-#### Configure your IBMQ credentials
+#### Configure your IBM Quantum credentials
 
-1. Create an _[IBM Q](https://quantum-computing.ibm.com) > Account_ if you haven't already done so.
-
-2. Get an API token from the IBM Q website under _My Account > API Token_.
+1. Create an _[IBM Quantum](https://quantum-computing.ibm.com)_ account if you haven't already done so.
 
+2. Get an API token from the IBM Quantum website under _[Account settings](https://quantum-computing.ibm.com/account) > API Token_.
+<img src="docs/images/ibm_account1.png" width="30%"/><img src="docs/images/ibm_account2.png" width="40%"/>
```

Comment:
You probably want to put a bit of horizontal space between these two images as well.

If possible, it would be neater if we could rescreenshot the two to:
- avoid the white background encroaching in the account dropdown
- make both images the same natural height (i.e. the same number of pixels tall), even if their widths are a bit different.

They'll sit next to each other on the page a bit more neatly if so.  These images also won't show up on the PyPI page, and they might be a bit temperamental on GitHub too because of the relative address.  One trick I've used in the past is to upload the images into a random issue, then use the permalink to those images in the readme instead.  The images don't end up checked into the repo, but the only way anyone would really see them anyway is on some online rendered version.

As an example, that's how the images in [this repository's readme](https://github.com/openqasm/openqasm.vim) are hosted - they're in openqasm/openqasm#239 (it was before we split the organisation off, so everything was still in a monorepo then).

Reply:
updated in 5cb64e623